Dockerfile

```diff
@@ -7,5 +7,10 @@ RUN curl https://sh.rustup.rs -sSf | bash -s -- -y
 RUN apt-get update && apt-get install build-essential -y
 RUN apt-get install binutils-dev
 RUN apt-get install libunwind-dev -y 
+RUN apt install git -y
 RUN cargo install honggfuzz
+RUN cargo install cargo-fuzz
+RUN rustup toolchain install nightly
+RUN git clone https://github.com/lambdaclass/fuzzing_examples.git
```

Comment:
This step should probably be run in a new layer, so that when working in development, the base image already has the updates, honggfuzz and cargo-fuzz installed, all you only have to wait for the clone and build.

Reply:
Just in case, this is called a multistage build https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#use-multi-stage-builds